Source/DataSources/CzmlDataSource.js

```diff
@@ -600,7 +600,10 @@ define([
         var hasInterval = defined(combinedInterval) && !combinedInterval.equals(Iso8601.MAXIMUM_INTERVAL);
 
         if (!isReference) {
-            referenceFrame = defaultValue(ReferenceFrame[packetData.referenceFrame], undefined);
+            if (defined(packetData.referenceFrame)) {
+                referenceFrame = ReferenceFrame[packetData.referenceFrame];
+            }
+            referenceFrame = defaultValue(referenceFrame, ReferenceFrame.FIXED);
```

Comment:
OK, so if you put in an invalid value, you quietly get FIXED without seeing any error.  I'm fine with that, but there are other places in CZML that could be likewise relaxed.  For example, if CZML can't find or fails to load a glTF, currently we throw and break the render loop, when really I think we should log an error and keep rendering the rest of the CZML.  That's out of scope for this PR of course, just making a note here that I might reference someday.